### DISCUSSION TITLE
Cambio modalità inserimento valutazioni

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#375] [Idee e proposte]
Sarebbe possibile per cortesia cambiare la fase di inserimento dei voti sia per quanto riguarda le verifiche orali,scritte e pratiche?Lo chiedo perchè non è più possibile ogni volta trascinare il cursore ,è parecchio fastidioso anche perchè non è sempre preciso , soprattutto quando si deve dare una mezza valutazione o comunque una valutazione includente i segni +,- .Pertanto propongo ,se è possibile di cambiare ,invece di questa striscia in cui si trascina il cursore per definire il voto,mettere semplicemente delle celle vuote ,dove cliccandoci dentro si inserisce la valutazione manualmente(come in Axios) ,oppure un menu a tendina dove cliccandoci sopra si sceglie la valutazione (come in Argo,classeviva).Lo stesso dicasi in fase di scrutinio

**Comment by trinko** [comment]
Anzi tutto segnalo che non è necessario trascinare il cursore: se fai un click sul cursore per attivarlo, poi puoi usare la tastiera con le frecce destra e sinistra, cosa che permette una maggiore precisione.

In futuro prevedo di fare modifiche. Ma non nel breve periodo.
